tests/integration/test_integration.py

```diff
@@ -26,7 +26,6 @@
 
 
 @pytest.fixture(scope="module")
-@pytest.mark.abort_on_fail
```

Comment:
why remove this?

Reply:
Markers on fixtures are deprecated since [pytest 7.4](https://docs.pytest.org/en/stable/deprecations.html#applying-a-mark-to-a-fixture-function)